Cargo.toml

```
@@ -27,6 +27,13 @@ edition = "2021"
 rust-version = "1.74"
 
 [dependencies]
+#!! Downgrade to versions still supporting v1.74.0
+#!! Make sure to remove on `next`
+home = "=0.5.9"
+litemap = "=0.7.4"
+zerofrom = "=0.1.5"
+idna = "=1.0.2"
```

```suggestion
litemap = "=0.7.4"
zerofrom = "=0.1.5"
```
These should suffice, and lets serenity compile on 1.74.